Can not use gccgo to compile .go file.

Hi All,

I found if I use any reflect2 function in my app, I can't use go build --compiler=gccgo to compile it.
It will return error like that:

go build --compiler=gccgo map_1.go 
# command-line-arguments
/tmp/go-build467403193/b023/_pkg_.a(_go_.o): In function `github_com_modern_go_reflect2.loadGo15Types':
/usr/lib/go-1.10/src/github.com/modern-go/reflect2/type_map.go:42: undefined reference to `reflect.typelinks'
/tmp/go-build467403193/b023/_pkg_.a(_go_.o): In function `github_com_modern_go_reflect2.loadGo17Types':
/usr/lib/go-1.10/src/github.com/modern-go/reflect2/type_map.go:74: undefined reference to `reflect.typelinks'
/usr/lib/go-1.10/src/github.com/modern-go/reflect2/type_map.go:78: undefined reference to `reflect.resolveTypeOff'
collect2: error: ld returned 1 exit status

Any idea to fix it?

Thanks
Zhang Chen

--- a/type_map.go
+++ b/type_map.go
@@ -1,3 +1,5 @@
+// +build !gccgo
+
 package reflect2
 
 import (

